sshd/zenith/sshd/tunnel.py

```diff
@@ -294,13 +299,12 @@ def consul_register_service(server_config, tunnel, subdomain):
     )
     # If we failed to register the service then bail
     if 200 <= response.status_code < 300:
-        print("[SERVER] [INFO] Registered service successfully")
+        logger.info("Registered service with Consul")
     else:
-        print("[SERVER] [ERROR] Failed to register service", file = sys.stderr)
-        sys.exit(1)
+        logger.critical("Failed to register service with Consul")
```

Comment:
Don't we need to raise an exception here?

Reply:
And again